src/sys/epoll.rs

```diff
@@ -1,66 +1,126 @@
-use std::os::unix::io::RawFd;
+use std::os::unix::{io::RawFd, prelude::AsRawFd};
```

Comment:
Why import this from the prelude instead of doing

```rust
use std::os::unix::io::{RawFd, AsRawFd};
```

?

Reply:
Well, we shall thank rust-analyzer's auto-fill. :sweat_smile: